bin/powershell/metric-windows-cpu-load.ps1

```diff
@@ -38,27 +38,16 @@ if ($UseFullyQualifiedHostname -eq $false) {
     $Path = [System.Net.Dns]::GetHostEntry([string]"localhost").HostName.toLower()
 }
 
-$perfCategoryID = Get-PerformanceCounterByID -Name 'Processor Information'
-$localizedCategoryName = Get-PerformanceCounterLocalName -ID $perfCategoryID
-
-$counters =  New-Object System.Collections.ArrayList
-
-[void]$counters.Add('% Processor Time')
-[void]$counters.Add('% Idle Time')
-[void]$counters.Add('% User Time')
-[void]$counters.Add('% Interrupt Time')
-
-foreach ($counter in $counters) {
+$Time = DateTimeToUnixTimestamp -DateTime (Get-Date)
 
-$perfCounterID = Get-PerformanceCounterByID -Name $counter
-$localizedCounterName = Get-PerformanceCounterLocalName -ID $perfCounterID
-$value = [System.Math]::Round((Get-Counter "\$localizedCategoryName(_total)\$localizedCounterName" -SampleInterval 1 -MaxSamples 1).CounterSamples.CookedValue)
+$value=(Get-CimInstance -ClassName Win32_PerfFormattedData_PerfOS_Processor -Filter "Name like '_Total'").PercentProcessorTime
```

Comment:
previously this accounted for localizing the counter name I dont think select average will work if windows is not set to english.